src/components/pages/TopPage/hooks.ts

```diff
@@ -2,7 +2,7 @@ import { useEffect } from "react";
 import { useNavigate } from "@reach/router";
 
 import { useAuth, useCleanUrl } from "@reearth/auth";
-import useNotification from "@reearth/notifications/hooks";
+import useNotification from "@reearth/components/organisms/Notification/hooks";
```

Comment:
```suggestion
import { useNotification } from "@reearth/state";
```

`@reearth/components/organisms/Notification/hooks` cannot be used directly by external components.

Directly using hooks of the component by external components complicates the dependencies between components and is therefore prohibited. 

Don't forget that we have gone to the trouble of introducing jotai to prevent this from happening. Otherwise, jotai itself is unnecessary.

Reply:
This is actually my poor naming. `useNotification` here is the name I gave for the notification-related hooks. As you can see below on line 16 I pull out notify, which is the function being used to update the notification (with checks and adding the notificationHeading).
Is there a more appropriate name I should use? I didn't think calling it `useHooks` was appropriate since that is the name we use for local hooks use.
Let me know if there is a better name since useNotification can get confused with Jotai.

Reply:
I think it's not about the name, it's about the fact that notifications from each page and organisms should always be updated via jotai, and useHooks should not be imported from any component other than organims/Notification. That hooks used internally.